monai/bundle/config_parser.py

```
@@ -35,6 +36,189 @@
 _default_globals = {"monai": "monai", "torch": "torch", "np": "numpy", "numpy": "numpy"}
 
 
+def _identity(value: Any) -> Any:
+    """Module-level reconstructor used by ``_ConfigProxy.__reduce__`` so proxies pickle as their raw value."""
+    return value
+
+
+def _wrap_parsed(parser: ConfigParser, id: str, value: Any) -> Any:
```

Ruff (0.15.13)

[error] Function argument `id` is shadowing a Python builtin

(A002)